<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgBalanceList</class>
 <widget class="QDialog" name="CDlgBalanceList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lava2res.qrc">
    <normaloff>:/images/res/icon.ico</normaloff>:/images/res/icon.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeitraum:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>vom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>bis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Artikel:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBrowseArticle">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Artikelauswahl</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>961</width>
     <height>371</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>Datum</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Anzahl(Eingang)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Anzahl(Ausgang)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Differenz</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bestand davor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bestand danach</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Artikelsaldenliste:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>540</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>drucken</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditFrom">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2009</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditTo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumDate">
    <date>
     <year>2009</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd.MM.yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetArticle">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>591</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>dateEditFrom</tabstop>
  <tabstop>dateEditTo</tabstop>
  <tabstop>listWidgetArticle</tabstop>
  <tabstop>pushButtonBrowseArticle</tabstop>
  <tabstop>tableWidgetList</tabstop>
  <tabstop>pushButtonPrint</tabstop>
 </tabstops>
 <resources>
  <include location="lava2res.qrc"/>
 </resources>
 <connections/>
</ui>
